codegen/src/test/java/io/improbable/keanu/codegen/python/GetNonExportedVertices.java

```diff
@@ -0,0 +1,34 @@
+package io.improbable.keanu.codegen.python;
+
+import io.improbable.keanu.annotation.ExportVertexToPythonBindings;
+import io.improbable.keanu.vertices.Vertex;
+import org.reflections.Reflections;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+import java.util.Set;
+
+public class GetNonExportedVertices {
```

Comment:
I guess we are not running this in tests to avoid being to strict? Useful tool!

Reply:
Yeah - at the moment,  we don't enforce anything about all vertices having to be exported to Python like we do for Saveability.  It certainly helped when it came time to annotate the extra ones we wanted to export though!